libs/core/src/badge/badge.element.ts

```diff
@@ -0,0 +1,45 @@
+import { CSSResultGroup, html } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import KirbyElement from '../internal/kirby-element';
+import baseStyles from '../styles/element.base.styles';
+
+import styles from './badge.element.styles';
+
+export type BadgeVariant = 'white' | 'success' | 'warning' | 'danger';
```

Comment:
Do we want themeColor to be named variant down the line? If not, should we consider calling this `BadgeThemeColor` for now?